modules/repository-s3/src/main/java/org/elasticsearch/repositories/s3/S3Repository.java

```diff
@@ -202,6 +203,26 @@ class S3Repository extends MeteredBlobStoreRepository {
         Setting.Property.Dynamic
     );
 
+    /**
+     * We will retry deletes that fail due to throttling. We use an {@link BackoffPolicy#linearBackoff(TimeValue, int, TimeValue)}
+     * with the following parameters
+     */
+    static final Setting<TimeValue> RETRY_THROTTLED_DELETE_DELAY_INCREMENT = Setting.timeSetting(
+        "throttled_delete_retry.delay_increment",
+        new TimeValue(50, TimeUnit.MILLISECONDS),
+        new TimeValue(0, TimeUnit.MILLISECONDS)
+    );
+    static final Setting<TimeValue> RETRY_THROTTLED_DELETE_MAXIMUM_DELAY = Setting.timeSetting(
+        "throttled_delete_retry.maximum_delay",
+        new TimeValue(500, TimeUnit.MILLISECONDS),
```

Comment:
`500ms` feels a little short for my taste, I'd expect something more like `5s` here.

Reply:
Fixed in 8868f2b5f1fa4ae9df84ab67e4abdf558ca8da40